src/helpers/ens.ts

```diff
@@ -507,7 +511,9 @@ const setupMulticallRecords = (
   if (textAssociatedRecord) {
     data.push(
       textAssociatedRecord
-        .filter(textRecord => Boolean(textRecord.value))
+        .filter(
+          textRecord => Boolean(textRecord.value) || textRecord.value === ''
```

Comment:
i feel like this one was also affecting users that erased a record manually in the ens assign records form
this is filtering out those empty string records

Reply:
good catch